policyengine_us/parameters/gov/states/nh/tax/credits/education_tax_credit/rate.yaml

```
@@ -0,0 +1,21 @@
+description: New Hampshire matches this share of Education Tax Credit, depending on the donation.
+metadata:
+  threshold_unit: currency-USD
+  rate_unit: /1
+  threshold_period: year
+  type: marginal_amount
```

I'm surprised `marginal_amount` worked; @nikhilwoodruff if you specify a `rate_unit: /1` does that make it a marginal rate?
```suggestion
  type: marginal_rate
```

---

No, marginal amount scales are their own scale type defined differently to marginal rate scales. The rate_unit is purely metadata and doesn't affect logic. Seems like it should be marginal rate instead?